expat/CMakeLists.txt

```diff
@@ -469,7 +469,7 @@ if(WIN32 AND NOT MINGW)
     #       Everything but MSVC is already adding prefix "lib", automatically.
     # NOTE: "set_property(TARGET expat PROPERTY PREFIX lib)" would only affect *.dll
     #       files but not *.lib files, so we have to rely on property OUTPUT_NAME, instead.
-    #       Property CMAKE_*_POSTFIX still applies.
+    #       Property EXPAT_*_POSTFIX still applies.
```

Comment:
Since it says "Property" in the beginning, let's maybe use `*_POSTFIX` or [`<CONFIG>_POSTFIX`](https://cmake.org/cmake/help/latest/prop_tgt/CONFIG_POSTFIX.html) here, instead?

Reply:
Makes sense.
```suggestion
    #       Target property <CONFIG>_POSTFIX still applies.
```